freqtrade/edge/edge_positioning.py

```diff
@@ -359,11 +359,11 @@ def _process_expectancy(self, results: DataFrame) -> Dict[str, Any]:
         # Returning a list of pairs in order of "expectancy"
         return final
 
-    def _find_trades_for_stoploss_range(self, ticker_data, pair, stoploss_range):
-        buy_column = ticker_data['buy'].values
-        sell_column = ticker_data['sell'].values
-        date_column = ticker_data['date'].values
-        ohlc_columns = ticker_data[['open', 'high', 'low', 'close']].values
+    def _find_trades_for_stoploss_range(self, dataframe, pair, stoploss_range):
```

Comment:
i don't like the naming of `dataframe` ...
It's way too generic - i know the type - but the content could be "anything" ...

Reply:
it's same dataframe and the same parameter name for it, as used in the populate_* methods in the strategy -- so also called as "dataframe".

I would not rename it to something like "dataframe_with_signals", but rather the helpstring with the description of the parameter should later be added, along with type hints, which are also missing here. Not for this PR.